src/content/ui/adaptive-responsive/more-info.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Additional resources
```

Comment:
This page is indexed separately from the rest of the pages, so "Additional resources" might be a bit confusing if it showed up in a search engine.

Perhaps something like:

```suggestion
title: Adaptive & responsive design learning resources
```